app/src/pages/inside/launchesPage/modals/launchImportModal/launchImportModal.scss

```diff
@@ -0,0 +1,50 @@
+.dropzone {
+  padding: 20px 20px 25px 20px;
+  text-align: center;
+}
+
+.dropzone-wrapper {
+  background-color: #f9f9f9;
```

Comment:
Use color variables. Please follow the sequence of defining formatting css rules (https://learn.javascript.ru/css-format).